src/features/Admin/Bookings/Bookings.tsx

```diff
@@ -138,71 +163,92 @@ const handleClose = () => {
-                        <MoreVertIcon />
-                      </IconButton>
-                      <Menu
-                        anchorEl={anchorEl}
-                        open={Boolean(anchorEl && selectedBooking?._id === booking?._id)}
-                        onClose={handleClose}
-                      >
-                        <MenuItem
-                          onClick={() => showViewModal(booking?._id)}
+                  bookings.map((booking, index) => (
+                    <TableRow
+                      key={booking?._id}
+                      style={
+                        index % 2
+                          ? { background: "#f6f6f6" }
+                          : { background: "white" }
+                      }
+                    >
+                      <TableCell>{booking?.status}</TableCell>
+                      <TableCell>{booking?.totalPrice}</TableCell>
+                      <TableCell>
+                        {new Date(booking?.startDate).toLocaleDateString()}
+                      </TableCell>
+                      <TableCell>
+                        {new Date(booking?.endDate).toLocaleDateString()}
+                      </TableCell>
+                      <TableCell>
+                        <IconButton
+                          onClick={(e) => handleMenuClick(e, booking)}
                         >
-                          <Tooltip title="View" arrow>
-                            <IconButton color="primary" >
-                              <VisibilityIcon  fontSize='small'/>
-                              <p style={{ fontSize: '14px' }}>View</p> 
-                              
-                            </IconButton>
-                          </Tooltip>
-                        </MenuItem>
-                     
-                        <MenuItem onClick={() => showDeleteModal(booking?._id)}>
-                          <Tooltip title="Delete" arrow>
-                            <IconButton color="error" >
-                              <DeleteIcon fontSize='small'/>
-                              <p style={{ fontSize: '14px' }}>Delete</p>
-                            </IconButton>
-                          </Tooltip>
-                        </MenuItem>
-                      </Menu>
-                    </TableCell>
-                     
-                      </TableRow>
-                    )
-                  )
+                          <MoreVertIcon />
+                        </IconButton>
+                        <Menu
+                          anchorEl={anchorEl}
+                          open={Boolean(
+                            anchorEl && selectedBooking?._id === booking?._id
+                          )}
+                          onClose={handleClose}
+                        >
+                          <MenuItem onClick={() => showViewModal(booking?._id)}>
+                            <Tooltip title="View" arrow>
```

Comment:
Tooltip component is responsible for showing a tip about the button that we are using when hovering.
If you hover over any menu item, it will show you the title passed to the tooltip.
This is beneficial when we only have icons, but in our case, we are explicitly writing what the icons are doing, so we can consider removing the tooltip around the menu items or removing the text beside the icon.

To be consistent, we can remove the paragraph text as in the Ads page.

Reply:
done

Reply:
about the facilities error samia fixed it in branch room-facility when it merge the error will disappear